R/expr-is.R

```diff
@@ -1,38 +1,68 @@
-#' Check whether a parse table corresponds to a certain expression
+#' @title Check whether a parse table corresponds to a certain expression
+#' @name pd_is
 #'
 #' @param pd A parse table.
-#' @name pd_is
+#' @param tilde_pos Integer vector indicating row-indices that should be
+#'   checked for tilde. See 'Details'.
+#'
+#' @family third-party style guide helpers
 #' @keywords internal
 NULL
 
-#' @describeIn pd_is Checks whether `pd` contains an expression wrapped in
-#'   curly brackets.
-#' @keywords internal
+#' @describeIn pd_is Checks whether `pd` contains an expression wrapped in curly brackets.
+#' @examples
+#' code <- "if (TRUE) { 1 }"
+#' pd <- compute_parse_data_nested(code)
+#' is_curly_expr(pd)
+#' is_curly_expr(pd$child$`17`$child$`14`)
+#'
```

Comment:
The blank lines are deliberate. 

Without them, all examples are squished together like a centipede, and it's hard to tell where one ends and the other starts.